### PR TITLE
Case Studies slices

### PR DESCRIPTION
**Pull request by AndrewBestbier**
## Motivation:
[Mobile Design](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57f667e34f06bb3a41a9385c)
[Desktop Design](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57f65fa5d0d4e073441c9c3a)

### General
- [x] Design Review
- [x] All links working
- [x] Firefox
- [x] Safari
- [x] IE
- [x] Fix funny thing with description text
- [x] Fix funny naming of heading/header Container

### Header
[Desktop](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57fb9d3a9f25e8402d7085e5)
[Mobile](https://app.zeplin.io/project.html#pid=57c8297fc3cf63795b6a4520&sid=57fb54f11b59061c2f2bd0ba)
- [x] Padding on Desktop 
- [x] Padding on Tablet
- [x] Padding on Mobile
- [x] Text Size on Desktop
- [x] Text Size on Tablet
- [x] Text Size on Mobile

### Proof in the pudding 
- [x] Padding on Desktop
- [x] Padding on Tablet
- [x] Padding on Mobile
- [x] Text Size on Desktop
- [x] Text Size on Tablet
- [x] Text Size on Mobile

### Fortnum & Mason
- [x] Padding on Desktop
- [x] Padding on Tablet
- [x] Padding on Mobile
- [x] Text Size on Desktop
- [x] Text Size on Tablet
- [x] Text Size on Mobile

### Camden Market
- [x] Padding on Desktop
- [x] Padding on Tablet
- [x] Padding on Mobile
- [x] Text Size on Desktop
- [x] Text Size on Tablet
- [x] Text Size on Mobile

### Financial Times
- [x] Padding on Desktop
- [x] Padding on Tablet
- [x] Padding on Mobile
- [x] Text Size on Desktop
- [x] Text Size on Tablet
- [x] Text Size on Mobile

### CODE REVIEW

**Comment by asavin** [comment]
Whoa those checkboxes

File: site/pages/case-studies/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classnames from 'classnames/bind';
+import styles from './style.css';
+import TechSlice from '../../slices/tech-slice';
+
+/* PNG and JPG  imports */
+import camdenJPG from './JPG/camden.jpg';
+import financialTimesJPG from './JPG/financialtimes.jpg';
+import fortnumJPG from './JPG/fortnum.jpg';
```
Comment (lpil) [inline site/pages/case-studies/index.js]: What's with the capital letters in paths?

File: site/pages/case-studies/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        </h1>
+      </div>
+      <TechSlice />
+      <div className={styles.headerContainer}>
+        <h1 className={styles.mainHeader}>The proof is in the pudding.</h1>
+        <h2 className={styles.subHeader}>Read some of our case studies.</h2>
+      </div>
+      <div className={styles.caseStudyContainer}>
+        <a className={styles.imageLink} href="https://red-badger.com/our-work/case-study/fortnum-and-mason/">
+          <img alt="The logo of Fortnum & Masons" src={fortnumJPG} />
```
Comment (AndrewBestbier) [inline site/pages/case-studies/index.js]: Mason rather than Masons.

File: site/pages/case-studies/style.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+.pageHeaderContainer {
+  display: flex;
+  composes: dividerWhite from "../../css/_divider.css";
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: white;
+  padding-top: 30px;
+  padding-bottom: 65px;
```
Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: Should be 60 or 70

File: site/pages/case-studies/style.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+.caseStudyDescription {
+  color: black;
+  composes: fontXS from "../../css/typography/_fonts.css";
+  composes: sansSerif from "../../css/typography/_fonts.css";
+}
+
+.inverse {
+  background-color: black;
+  color: yellow;
```
Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: cainYellow!

File: site/pages/case-studies/style.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+  composes: fontXS from "../../css/typography/_fonts.css";
+  composes: boldSansSerif from "../../css/typography/_fonts.css";
+  font-weight: bold;
+  text-transform: uppercase;
+  margin-bottom: 5px;
+  width: 100%;
+}
+
+.imageLink {
+  font-size: 0;
```
Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: What's this?

File: site/pages/case-studies/style.css (hunk 4 above)
Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: wtf andrew

Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: I am andrew

File: site/pages/case-studies/style.css [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    display: flex;
+    padding: 60px 20px 60px 20px;
+  }
+
+  .caseStudyTextContainer {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    padding: 0px 10%;
+    flex:1 ;
```
Comment (AndrewBestbier) [inline site/pages/case-studies/style.css]: Space!

File: site/slices/tech-slice/styles.css [hunk 7]
```diff
@@ -17,17 +17,16 @@
 
 .heading {
   composes: widthContainer;
-  composes: serif from "../../../css/typography/_fonts.css";
-  composes: fontL from "../../../css/typography/_fonts.css";
+  composes: serif from "../../css/typography/_fonts.css";
+  composes: fontL from "../../css/typography/_fonts.css";  text-align: center;
```
Comment (AndrewBestbier) [inline site/slices/tech-slice/styles.css]: Two rules on one line.

File: site/slices/tech-slice/styles.css [hunk 8]
```diff
@@ -57,26 +56,26 @@
...
   }
 
   .heading {
     margin-bottom: 30px;
   }
 
   .projectsBlerb {
-    composes: fontSerif from "../../../css/typography/_fonts.css";
-    composes: fontS from "../../../css/typography/_fonts.css";
+    composes: fontSerif from "../../css/typography/_fonts.css";
```
Comment (AndrewBestbier) [inline site/slices/tech-slice/styles.css]: Should be serif.

Comment (AndrewBestbier) [inline site/slices/tech-slice/styles.css]: Can remove this apparently.

**Pull request merged**